rome-osgi-test/pom.xml

```diff
@@ -11,6 +10,43 @@
     </parent>
 
     <artifactId>rome-osgi-test</artifactId>
+    <packaging>jar</packaging>
+
+    <name>rome-osgi-test</name>
+
+    <description>OSGi tests for ROME</description>
```

Comment:
It is required for publishing to Maven Central to set the name and description. In this case it can be removed because rome-osgi-test will never be published to Maven central.